presenter-in-iframe/index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>FieldTwin in iframe</title>
+    <style>
+      body {
+        background-color: #ffffff;
+      }
+    </style>
+  </head>
+  <body style="height: 100vh; margin: 0; border: 0">
+    <div style="width: 100%; height: 100%; display: flex; overflow: hidden">
+      <div style="width: 30%; height: 100%; display: flex; flex-flow: column">
+        <div style="width: 100%; flex: 0 1 auto">
+          <button onclick="getProjectData()">
+            <strong> Get data for project </strong>
+          </button>
+        </div>
+        <div style="width: 100%; flex: 0 1 auto">
+          <button onclick="computeCostUsingServer()">
+            <strong> Get Cost from Cost server </strong>
```

Comment:
This might need some explanation that it only works if a cost server is deployed